lib/config/options/index.ts

```diff
@@ -852,7 +852,7 @@ const options: RenovateOptions[] = [
   {
     name: 'matchBaseBranches',
     description:
-      'List of branches to match (e.g. ["master"]). Valid only within `packageRules` object.',
+      'List of branches to match (e.g. ["master"]) or list of regex patterns (e.g. ["/^release\\/.*/"]). Valid only within `packageRules` object.',
```

Comment:
```suggestion
      'List of branches to match (e.g. `["master"]`) or list of regex patterns (e.g. `["/^release\\/.*/"]`). Valid only within `packageRules` object.',
```

This makes the final text look like this, which should be easier to read. 😄 

> List of branches to match (e.g. `["master"]`) or list of regex patterns (e.g. `["/^release\\/.*/"]`). Valid only within `packageRules` object.

Reply:
They can also be mixed, so it should be a list of branches or regex patterns

Reply:
> They can also be mixed, so it should be a list of branches or regex patterns

Can you suggest a better `description`? I'm not sure what you want me to change... 🙈

Reply:
Maybe something linke this?
```
List of strings containing exact matches (e.g. `["master"]`) and/or regex expressions (e.g. `["/^release\\/.*/"]`). Valid only within `packageRules` object.
```

Reply:
I like it! 😄